### DISCUSSION TITLE
custom rule and severity level

### DISCUSSION THREAD

**Discussion by morohyahaya** [Security-Onion-Solutions/securityonion#10824] [Unsupported Versions]
hello I need help is it possible to modify the alert level and the suricata rules if so how to personalize the suricata rules and how to modify the criticiter level and especially the file modification steps I have looked at all the discussions I have not found a solution to my problem .

**Comment by robbiemarshall** [comment]
Documentation on modifying Suricata rules can be found here: https://docs.securityonion.net/en/2.3/managing-alerts.html

If you need to create a new Suricata rule, you can go here: https://docs.securityonion.net/en/2.3/local-rules.html#nids

Additional documentation on Suricata can be found here:  https://docs.suricata.io/en/latest/
